frappe/utils/backups.py

```diff
@@ -352,12 +352,21 @@ def backup_files(self):
 			else:
 				cmd_string = "tar -cf {0} {1}"
 
-			frappe.utils.execute_in_shell(
-				cmd_string.format(backup_path, files_path),
-				verbose=self.verbose,
-				low_priority=True,
-				check_exit_code=True,
-			)
+			try:
+				frappe.utils.execute_in_shell(
+					cmd_string.format(backup_path, files_path),
+					verbose=self.verbose,
+					low_priority=True,
+					check_exit_code=True,
+				)
+			except frappe.CommandFailedError as e:
+				if e.err and "file changed as we read it" in e.err:
+					click.secho(
+						"Ignoring `tar: file changed as we read it` to prevent backup failure",
+						fg="red",
+					)
```

Comment:
You probably can't ignore this if you're gzipping?

Reply:
Yeah didn't try gzip, will see if it happens.
<hr>
Hmm, I've been testing locally by creating a bunch of files while the backup process is running, either case shouldn't break a backup, seems like 1.30 on FC.